invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -256,11 +254,17 @@ class CommunityProfileForm extends Component {
       },
     });
 
-    // Deserialize organizations
-    let organizations = _map(
-      _get(initialValues, "metadata.organizations"),
-      "name"
-    );
+    // Deserialise organisations. Store known communities.
```

Comment:
This needs to be cross-checked with @ntarocco

Reply:
After discussions, we don't see a better solution